presto-yugabyte/pom.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.prestosql</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>346-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-yugabyte</artifactId>
+    <description>Presto - Yugabyte Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.prestosql</groupId>
+            <artifactId>presto-cassandra</artifactId>
+            <version>346-SNAPSHOT</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.facebook.presto.cassandra</groupId>
```

Comment:
Do we need this exclusion?

Reply:
This was according to @findepi suggestion in #5352

Reply:
You should exclude
```
<groupId>io.prestosql.cassandra</groupId>
<artifactId>cassandra-driver</artifactId>
```
instead.